drivers/blktap2.py

```diff
@@ -2239,7 +2243,7 @@ def __init__(self, domid, devid):
         self.devid = int(devid)
         self._xbt = XenbusDevice.XBT_NIL
 
-        import xen.lowlevel.xs
+        import xen.lowlevel.xs  # pylint: disable-import-error
```

Comment:
that should be `disable=import-error`, but is likely made redundant by the global setting in config file

Reply:
ah, that's why that didn't work :( a simple typo, bah.